doc/user/INTERACTIONS.md

```diff
@@ -48,12 +48,18 @@ Other options can be toggled directly by pressing the following hotkeys:
 
 Note that some hotkeys can be available or not depending on the file being loaded and the F3D configuration.
 
+Camera Hotkeys:
+* `1`: front view camera.
+* `3`: right view camera.
+* `7`: top view camera.
+* `9`: isometric view camera.
```

Comment:
do you expect other number to be used for camera in the future ? What behavior will it be ?

Reply:
We could make it similar to Blender so it's familiar with Blender users, but also with some minor changes:
- `1, 3 and 7`: Axis movement (Front, Right, Top View);
- `9`: Front isometric view; 
- `Shift`, `Alt` or `Ctrl` modifiers: Invert views (Rear, Left, Bottom, Rear Isometric);
- `5`: Orthographic toggle.
- `4` and `6`: rotate camera left and right;
- `2` and `8` rotate camera top or bottom;

Reply:
thanks for sharing, good start in this PR anyway :)

Reply:
I also forgot to mention that we could also be able to snap to axis movement using `Ctrl + mouse camera pan`.

Reply:
another feature entirely, please open an issue :)

Reply:
Should I open 2 issues? One new issue addressing the updated camera shortcuts feature and another one addressing the `Ctrl + mouse camera rotate` feature?